Library/Homebrew/dev-cmd/update-python-resources.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/pypi"
+
+module Homebrew
+  module_function
+
+  def update_python_resources_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update-python-resources` [<options>] <formula>
+
+        Update versions for PyPI resource blocks in <formula>.
+      EOS
+      switch "-p", "--print-only",
+             description: "Print the updated resource blocks instead of changing <formula>."
+      switch "-s", "--silent",
+             description: "Suppress any output."
+      min_named :formula
+    end
+  end
+
+  def update_python_resources
+    update_python_resources_args.parse
+
+    # Ensure python@3.8 and pipgrip are installed
+    unless Formula["python@3.8"].any_version_installed?
+      odie '"python@3.8" must be installed (`brew install python@3.8`)'
```

Comment:
Is it really needed to check for python@3.8 since it is a dependency of pipgrip? Installed pipgrip formula means installed python.

Reply:
Good point 👍